tools/evaluation.py

```diff
@@ -182,7 +213,15 @@ def main():
                                   basic_table_info, args.batch_size,
                                   args.samples_path, **args.sample_cfg)
     else:
-        raise NotImplementedError("We hasn't implemented multi gpu eval yet.")
+        find_unused_parameters = cfg.get('find_unused_parameters', False)
+        model = MMDistributedDataParallel(
+            model.cuda(),
+            device_ids=[torch.cuda.current_device()],
+            broadcast_buffers=False,
+            find_unused_parameters=find_unused_parameters)
+        single_gpu_online_evaluation(model, data_loader, metrics, logger,
```

Comment:
I just find the name of this function should be revised. `single_gpu` seems redundant and not correct.

Reply:
In addition, it seems that you have not considered saving images in DDP. Can we support it? Sampling and saving images in DDP may save more time.